bit-json-editor.mjs

```diff
@@ -1,89 +1,100 @@
-import { Component, Reflect as canReflect } from "//unpkg.com/can@5/core.mjs";
+import {
+	Reflect as canReflect,
```

Comment:
`canReflect` is exported so you don’t have to write `Reflect as canReflect`

Reply:
the cake is a lie